stablehlo/reference/Tensor.cpp

```diff
@@ -93,6 +93,11 @@ Tensor::Tensor(TensorType type)
 Tensor::Tensor(TensorType type, AsmResourceBlob blob)
     : impl_(llvm::makeIntrusiveRefCnt<detail::Buffer>(type, std::move(blob))) {}
 
+Tensor::Tensor(TensorType type, const Element &element)
+    : impl_(llvm::makeIntrusiveRefCnt<detail::Buffer>(type)) {
+  this->set({}, element);
+}
```

Comment:
I see how this constructor is more convenient that first creating a tensor, and then doing a set. 

However, I think that it's currently not pulling its weight. On the one hand, it's used only once, and its convenience is marginal (compresses a 2-liner into a 1-liner). On the other hand, it does introduce a new function and complicate the API of `Tensor`, so the cost is non-trivial.

Let's inline the constructor into the callsite?